sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -75,10 +75,23 @@ static void usb_receipt_callback_1(uint8_t c) {
   ++usb_chars_recved_total;
 }
 
+static void usb_send_str(char *s, usb_ss_ctx_t *ss) {
+  while (*s) usb_simpleserial_send_byte(ss, *s++);
+}
+
 static dif_gpio_t gpio;
 
+#define PINFLIP 0
+#define DIFFIO 0
+
+// pre-processor magic, MKSTR will put arg in quotes
+// (Can't just use #p in PHY because it does not expand the define)
+#define MKSTR(s) #s
+#define PHY(p, d) "pinflip=" MKSTR(p) " differential io = " MKSTR(d)
```

Comment:
Do not add unnecessary preprocessor magic to the SW tree. Use format specifiers like `%d`, directly in `LOG_INFO`, instead.

Reply:
Too used to the original library where we didn't have that! Fixed.